opencti-platform/opencti-front/eslint-plugin-custom-rules/lib/rules/classes-rule.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  meta: {
+    type: "problem",
+    fixable: "code",
+    docs: {
+      description: "MakeStyles should define only the classes used in the component"
+    },
+  },
+  create: (context) => {
+    const sourceCode = context.getSourceCode();
+    return {
+      ArrowFunctionExpression(node) {
+        if (node.parent.callee && node.parent.callee.name === "makeStyles") {
+          if (!node.body.properties) {
+            if (!node.body.body) {
+              context.report({
+                node: node,
+                message: "MakeStyles is used with no class declared",
+                data: {
+                  identifier: node.name
+                }
+              });
+            }
+          } else {
+            node.body.properties.forEach((v) => {
+              if (!sourceCode.text.includes("classes." + v.key.name)) {
```

Comment:
Est-ce qu'on ne mettrait pas par écrit dans le doc Best Practice sur Notion qu'il faut utiliser le mot clé `classes` du coup ? Sinon la règle sera bypass

Reply:
We could write it but I think everyone (that has already done some front) knows that we have that convention :)

Reply:
It's a possibility that yesterday I was close to use `styles` as a keyword... 😶 But yep I understand what you mean